src/main/java/com/minecolonies/coremod/colony/ColonyView.java

```diff
@@ -1211,6 +1211,12 @@ public void increaseBoughtCitizenCost()
 
     }
 
+    @Override
+    public List<EntityPlayer> getImportantMessageEntityPlayers()
+    {
+        return null;
+    }
```

Comment:
maybe we make this here an empty list so we can @NotNull the entire method?

Reply:
ye